cypress/support/awx-commands.ts

```diff
@@ -417,15 +417,18 @@ Cypress.Commands.add('deleteAwxUser', (user: User) => {
 
 Cypress.Commands.add(
   'createAwxProject',
-  (project?: SetRequired<Partial<Omit<Project, 'id'>>, 'organization'>, skipSync?: boolean) => {
+  (
+    project?: SetRequired<Partial<Omit<Project, 'id'>>, 'organization'>,
+    options?: { skipSync?: boolean }
+  ) => {
     cy.awxRequestPost<Partial<Project>, Project>('/api/v2/projects/', {
       name: 'E2E Project ' + randomString(4),
       scm_type: 'git',
       scm_url: 'https://github.com/ansible/ansible-ui',
       ...project,
     }).then((project) => {
-      if (!skipSync) {
-        waitForProjectToFinishSyncing(project.id);
+      if (!options?.skipSync) {
```

Comment:
I thing this set up is going to bite us.  This is an anti-pattern in controller as a project cannot be used anywhere until it is synced.  At a minimum we should have skipSync default to false.

Reply:
it does default to false